soundata/datasets/fsdnoisy18k.py

```diff
@@ -0,0 +1,319 @@
+
+"""
+
+import os
+from typing import BinaryIO, Optional, Tuple
+
+import librosa
+import csv
+import numpy as np
+
+from soundata import download_utils, jams_utils, core, annotations, io
+
+
+BIBTEX = """
+@misc{fonseca2019learning,
+      title={Learning Sound Event Classifiers from Web Audio with Noisy Labels},
+      author={Eduardo Fonseca and Manoj Plakal and Daniel P. W. Ellis and Frederic Font and Xavier Favory and Xavier Serra},
+      year={2019},
+      eprint={1901.01189},
+      archivePrefix={arXiv},
+      primaryClass={cs.SD}
+}
+"""
+REMOTES = {
+    "audio_train": download_utils.RemoteFileMetadata(
+        filename="FSDnoisy18k.audio_train.zip",
+        url="https://zenodo.org/record/2529934/files/FSDnoisy18k.audio_train.zip?download=1",
+        checksum="34dc1d34ca44622af5bf439ceb6f0d55",
+    ),
+    "audio_test": download_utils.RemoteFileMetadata(
+        filename="FSDnoisy18k.audio_test.zip",
+        url="https://zenodo.org/record/2529934/files/FSDnoisy18k.audio_test.zip?download=1",
+        checksum="1ac73d70b4ef3f81900d98c261a832de",
+    ),
+    "docs": download_utils.RemoteFileMetadata(
+        filename="FSDnoisy18k.doc.zip",
+        url="https://zenodo.org/record/2529934/files/FSDnoisy18k.doc.zip?download=1",
+        checksum="093a1ca185ec341ca4eac14215e7f96b",
+    ),
+    "metadata": download_utils.RemoteFileMetadata(
+        filename="FSDnoisy18k.meta.zip",
+        url="https://zenodo.org/record/2529934/files/FSDnoisy18k.meta.zip?download=1",
+        checksum="96e27a4a63b7a2870522ddcedb5d8296",
+    ),
+}
+
+LICENSE_INFO = """
+Please note that FSDnoisy18k has licenses at two different levels. All sounds in Freesound are released
+under Creative Commons (CC) licenses, and each audio clip has its own license as defined by the audio clip
+uploader in Freesound. In particular, all Freesound clips included in FSDnoisy18k are released under either
+CC-BY or CC0. For attribution purposes and to facilitate attribution of these files to third parties, we
+include a relation of audio clips and their corresponding license in the LICENSE-INDIVIDUAL-CLIPS file
+downloaded with the dataset.
+"""
+
+
+class Clip(core.Clip):
+    """FSDnoisy18K Clip class
+
+    Args:
+        clip_id (str): id of the clip
+
+    Attributes:
+        audio_path (str): path to the audio file
+        clip_id (str): clip id
+
+    Properties:
+        tag (soundata.annotations.Tags): tag (label) of the clip + confidence
+        aso_id (str): the id of the corresponding category as per the AudioSet Ontology
+        manually_verified (str): flag to indicate whether the clip belongs to the clean portion (1), or to the noisy portion (0) of the train set
+        noisy_small (str): flag to indicate whether the clip belongs to the noisy_small portion (1) of the train set
+        split (str): flag to indicate whether the clip belongs the train or test split
```

Comment:
Hey @genisplaja the description of these properties is making me think they are binary indicators, like `boolean` flags. Is it the case or are they labels e.g. "clean_portion" or "noisy_portion"?

Reply:
No, they are definitely a bool. My bad... I copy pasted the lines and forgot to update the data type. Sorry!

Reply:
Yes it's ok, if they are `int` or `float` (I meant no `str` XD) that should work as `boolean`